docs/user-documentation/access-control.md

```diff
@@ -146,9 +146,8 @@ Sites concerned with access control  wish to "lock down" their Fedora to only be
 
 ## Access control in the Triplestore
 
-In Islandora Defaults, there is no access control on the Blazegraph triplestore. Islandora defaults is configured so that all RDF triples sent to Fedora are also populated into the triplestore. If using the Ansible Playbook, it is available at http://localhost:8080/bigdata/. Like the solr index, it is open to the world by default.
-
+In a site created with ISLE or the Islandora Playbook, there is no access control on the Blazegraph triplestore. The Islandora Starter Site configuration is such that all RDF triples sent to Fedora are also populated into the triplestore. Like the solr index, it is open to the world by default.
```

Comment:
You're totally right. The [ISLE docs](https://github.com/Islandora-Devops/isle-dc#getting-started) say you can explore them at traefik.me URLs, but at least since [10 months ago](https://github.com/Islandora-Devops/isle-dc/commit/996b46cc2b7aa617f1aa3a2b8089268afbcdf9bf) they're not exposed.

Reply:
I hope I wasn't being to picky with the phrasing and your concern is absolutely valid too. I do like how you reworded it now. I only brought it up because the question of exposing those services has come up a few times in slack this year. It shouldn't be destructive to existing sites data to make the change but it may make the site temporarily unavailable in order to pull in the changes.